### DISCUSSION TITLE
MNIST classifier

### DISCUSSION THREAD

**Discussion by gautam-e** [mojodojodev/mojodojo.dev#26] [Ideas]
I guess this is the right place to put this discussion @mojodojodev (?).
As I said, I am working on an MNIST classifier and here's the stuff I have so far: https://gautam-e.github.io/slomojo/
I think I would benefit from the `numpy` example you are working on to improve what I have so far.

**Comment by gautam-e** [comment]
I'd be happy to ultimately put it up on mojodojo.dev once it reaches a certain maturity.

**Comment by mojodojodev** [comment]
This is a great idea! Looks great, let me know when I can link in this in This Week in Mojo, I might start an Awesome Mojo page with links to cool content like this as well
